xml/admin_monitoring_alerting.xml

```diff
@@ -494,129 +494,20 @@
   </note>
 
 <screen>
-service_type: container
-service_id: prometheus-webhook-snmp
+service_type: snmp-gateway
+service_name: snmp-gateway
 placement:
     <replaceable>ADD_PLACEMENT_HERE</replaceable>
-image: registry.suse.com/ses/7/prometheus-webhook-snmp:latest
-args:
```

Comment:
Is the prometheus-webhook-snmp image no longer required (or provided someplace else)?

Reply:
> Is the prometheus-webhook-snmp image no longer required (or provided someplace else)?

It will no longer be needed if we implement the following suggestion.

Ceph upstream has changed the OIDs of all [Prometheus alerts](https://github.com/ceph/ceph/commit/cfc4d3ec67292cc79e28d64582751e10fe5710af#diff-cf1112e777b119958d89b5464bda47ea828a2cb009729b0693d0d35cc0ec55c1). Because of that our SNMP webhook will not work correctly anymore without adaption or adaption of service spec in the documentation. To reduce the maintenance and prevent problems i would suggest to drop our own solution and advise our customers to switch to the official Ceph upstream SNMP gateway. This should be simple by deploying the new snmp-gateway service and delete the old one. This way the MIB and Prometheus alerts are always in sync.

We can adapt our own SNMP webhook, but this will require additional work and we always need to take care what upstream happens. Because of that i suggest to fully switch to the upstream solution; incl. customers that are using our own solution in SES7.

Reply:
this will deploy the upstream `docker.io/maxwo/snmp-notifier:v1.2.1` container image via cephadm.

We should probably create a downstream build of this image and update the cephadm image paths....

Reply:
@votdev Any progress or thoughts on a downstream snmp-notifier (as Mike suggested)?

Reply:
Well... the problem is not only about the container. Unfortunately we need a package of the upstream repository which is not available yet.